### ISSUE TITLE
iPad 7th gen not working

### ISSUE THREAD

**Issue by NatsuDzn** [MatthewPierson/checkm8-nonce-setter#11]
Hi, I have an error while using your tool with the iPad 7th gen (2019)
I know that it's available in the README but I can get it to work 
I have the following message : "ERROR: Unable to discover device model"
<img width="1039" alt="Capture d’écran 2019-12-03 à 19 52 55" src="https://user-images.githubusercontent.com/10818665/70080374-b295d480-1606-11ea-98f4-057543a7e9fb.png">

**Comment by MatthewPierson** [comment]
Is your device plugged into the computer in DFU mode?

**Comment by NatsuDzn** [comment]
> Is your device plugged into the computer in DFU mode?

Yes even the checkra1n app recognize that the iPad is in DFU Mode 
<img width="1680" alt="Capture d’écran 2019-12-07 à 18 19 36" src="https://user-images.githubusercontent.com/10818665/70378232-58568580-191e-11ea-9b96-34a5e8c9a5f8.png">
